lady_luck.js

```diff
@@ -146,21 +128,17 @@ function nwodRoll(pool, options) {
 }
 
 function suxxToWords(suxx) {
-	if (suxx == 1) {
-		return "1 success";
-	} else {
-		return `${suxx == 0 ? 'No' : suxx} successes`;
-	}
+	return suxx
+	  ? `${suxx} success${suxx == 1 ? '' : 'es'}`
+	  : 'No success'
 }
 
 function againToWords(again) {
-	if (again == 10) {
-		return '';
-	} else if (again < 10) {
-		return ` (${again}-Again)`;
-	} else {
-		return " (No rerolls)";
-	}
+	return again == 10
+	  ? ''
+	  : again < 10
+		? ` (${again}-Again)`
+		: " (No rerolls)"
```

Comment:
Nested ternaries are generally not a good idea. It renders the code less readable.